runtime/port/linuxs390/j9gs.c

```diff
@@ -114,4 +123,9 @@ j9gs_deinitialize(struct J9PortLibrary *portLibrary, struct J9GSParameters *gsPa
 	Trc_PRT_gs_deinitialize_Exit();
 
 	return IS_GS_INITIALIZED(gsParams) ? 0 : 1;
+
+#else /* !defined(J9ZTPF) */
```

Comment:
Same question as above.  Can we limit the ztpf-specific code paths by changing what's #ifdef'd out

Reply:
Update made.